app/core/redux/slices/notifications/index.ts

```diff
@@ -24,8 +24,12 @@ const slice = createSlice({
       state,
       action: PayloadAction<NotificationsAccountsState>,
     ) => {
-      if (JSON.stringify(state) !== JSON.stringify(action.payload)) {
-        return action.payload;
+      // Check if the payload is not an empty object
+      if (Object.keys(action.payload).length > 0) {
+        // Only update if there are differences
+        if (JSON.stringify(state) !== JSON.stringify(action.payload)) {
```

Comment:
What do you think about using the `isEqual` function of lodash instead? Any strong opinions on why JSON.stringify would be better? 

Interesting, I found a [benchmark](https://www.measurethat.net/Benchmarks/Show/4089/0/lodashisequal-vs-jsonstringify-equality-comparison-for#latest_results_block) that shows that the stringify comparison is quicker!

```suggestion
        if (isEqual(state, action.payload)) {
```

Reply:
Good change, but think that the correct change would be `if (!isEqual(state, action.payload)) {`. Changed